llm4pddl/approaches/llm_open_loop_approach.py

```diff
@@ -94,9 +104,18 @@ def _create_prompt(task: Task, plan: Optional[Plan] = None) -> str:
             obj_type = domain.constants[obj]
             type_to_objs[obj_type].append(obj)
         objects_strs: List[str] = []
+        # Create dictionary for randomizing object names.
+        objs_dict = {}
+        objs_list = []
+        for _, objs in type_to_objs.items():
+            objs_list += objs
```

Comment:
if the same object name appears multiple times in this list, we may have strange behavior. can we instead have `utils.randomize_object_names` take in a _set_ of objects?